plugins/filebrowser/tpl/editorbrowser.html

```diff
@@ -5,7 +5,7 @@
 <html lang="en">
     <head>
         <title>Filebrowser</title>
-        <link rel="stylesheet" href="<?php echo "{$pth['folder']['corestyle']}stylesheet.css"?>" type="text/css">
+        <link rel="stylesheet" href="<?php echo "{$pth['folder']['corestyle']}xhstyles.css"?>" type="text/css">
         <style type="text/css">body {padding: 0; margin: 0; overflow: auto;}</style>
```

Comment:
If it's something that should be fixed, it's fine to commit it right away (if anybody thinks otherwise, they still can complain, and the commit could be reverted). In this case, I'd suggest to do a separate pull request.